club/models.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Member(AbstractUser):
+    email = models.EmailField(unique=True)
+    type = models.CharField(max_length=20)
+    address = models.CharField(max_length=100)
+    home_phone = models.CharField(max_length=20)
+    mobile = models.CharField(max_length=20)
+    date_added = models.DateTimeField(auto_now_add=True)
+    
+    def __str__(self):
+        return f'{self.first_name} {self.last_name}'
+    
+    
+class Booking(models.Model):
+    IS_RECURRING_CHOICES = [(False, 'No'), 
+                            (True, 'Yes')]
+    time = models.DateTimeField()
```

Comment:
What is time here? Wouldn't that already be part of start_date?

Reply:
It's supposed to be the time of the booking, but I already have a start_date, which will have the time. So I will remove it